cws-service/src/main/java/jpl/cws/controller/RestService.java

```diff
@@ -311,40 +311,26 @@ public ModelAndView getInitiatorsHtmlTable() {
         	bufferedWriter.write(xmlData);
 		} catch (IOException e1) {
 			e1.printStackTrace();
+			return "ERROR: " + e1.getMessage();
 		} finally {
             try {
-                if (bufferedWriter != null) bufferedWriter.close();
+                if (bufferedWriter != null) {
+                	bufferedWriter.close();
+                }
             } catch(Exception e2) {
                  e2.printStackTrace();
             }
         }
-		FileInputStream input = null;
-		try {
-			input = new FileInputStream(f);
-		} catch (FileNotFoundException e3) {
-			// TODO Auto-generated catch block
-			e3.printStackTrace();
-		}
+
 		MultipartFile multipartFile = null;
```

Comment:
Not necessary to define outside try block, as it isn't used outside the `try`'s scope. Not critical.